### PR TITLE
Allow specifying SSL/TLS in fabric.connect() method

### PR DESCRIPTION
**Pull request by jooskim**
`FabricApi`'s connect() was only allowing insecure `ws://` websocket endpoints not `wss://`, but its underlying method does support connecting to SSL enabled endpoints. This PR exposes the ability to specify whether to connect to a secure wss:// STOMP endpoint.

Signed-off-by: Josh Kim <kjosh@vmware.com>

### CODE REVIEW

File: src/fabric.api.ts [hunk 1]
```diff
@@ -50,6 +51,7 @@ export interface FabricApi {
         host?: string,
         port?: number,
         endpoint?: string,
+        useSSL?: boolean,
```
Comment (daveshanley) [inline src/fabric.api.ts:54]: This means this is a breaking change as it introduces the optional arg in the middle of the signature. We will have to bump a feature version.

File: src/fabric/fabric.ts [hunk 2]
```diff
@@ -109,6 +109,7 @@ export class FabricApiImpl implements FabricApi {
         host?: string,
         port?: number,
         endpoint: string = '/fabric',
+        useSSL: boolean = false,
```
Comment (daveshanley) [inline src/fabric/fabric.ts:112]: will require a feature version bump.

**Review by daveshanley — APPROVED** [review]
This is a breaking change, so we need to go up a feature version.

File: src/fabric.api.ts (hunk 1 above)
Comment (jooskim) [reply src/fabric.api.ts:54]: actually that's a good point. i wanted to group arguments by similar/same areas but that essentially breaks the existing API.

**Pull request merged**